src/main/java/core/basesyntax/figure/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figure;
+
+public interface CalculateArea {
```

Comment:
```suggestion
public interface AreaCalculator {
```
please don't use verbs for class/interface names

Reply:
done